src/main/java/net/masterthought/cucumber/generators/EscapeHtmlReference.java

```diff
@@ -12,17 +12,17 @@
  */
 final class EscapeHtmlReference implements ReferenceInsertionEventHandler {
 
-    private static final PolicyFactory LINKS = new HtmlPolicyBuilder()
-            .allowStandardUrlProtocols().allowElements("a").allowAttributes("href")
-            .onElements("a").requireRelNofollowOnLinks().requireRelsOnLinks("noopener", "noreferrer")
+    private static final PolicyFactory TAGS = new HtmlPolicyBuilder()
+            .allowStandardUrlProtocols().allowElements("a", "span", "img").allowAttributes("href", "onclick", "src", "id", "style")
+            .onElements("a", "span", "img").requireRelNofollowOnLinks().requireRelsOnLinks("noopener", "noreferrer")
             .toFactory();
```

Comment:
Hello mpkorstanje,

embed - it's not a very good solution for highlighting text and highlighting warning messages. look at the screenshots please, there is highlighted text for test reproduction and a highlighted warning message

![2017-04-17_1526](https://cloud.githubusercontent.com/assets/20298105/25089560/44118c1e-238f-11e7-993e-a23d128451c0.png)
![2017-04-17_1656](https://cloud.githubusercontent.com/assets/20298105/25089562/4609a362-238f-11e7-9176-29a0840b1f76.png)